images/squid/Dockerfile

```diff
@@ -51,6 +51,14 @@ RUN systemctl enable squid.service
 # Workaround for https://github.com/moby/moby/issues/31243
 RUN usermod -a -G tty squid
 
+# Make rootCA.crt easy to access externally with:
+# nc localhost 9999
+COPY squid-root-ca.socket /etc/systemd/system/
+COPY squid-root-ca@.service /etc/systemd/system/
+RUN systemctl enable squid-root-ca.socket
+
+EXPOSE 9999/tcp
+
```

Comment:
Since this is to get the rootCA and user need to also add `-p 9999:9999` so better to use `podman exec` command.

```
$ podman run -d -p 3128:3128 -p 3129:3129 --name=squid 95605b5c1e47
$ podman exec squid cat /etc/pki/squid/rootCA.crt
-----BEGIN CERTIFICATE-----
[....]
-----END CERTIFICATE-----
```

Reply:
"better" depends on the usecase, you don't necessarily have easy access to the machine where the podman container is running (which is needed to use `podman exec`), while connecting to port 9999 can be done from anywhere. But I agree this port 9999 thing is just a "convenience hack", I don't mind dropping it.